evm/src/executor/inspector/cheatcodes/mod.rs

```diff
@@ -88,6 +92,22 @@ pub struct Cheatcodes {
 
     /// Additional, user configurable context this Inspector has access to when inspecting a call
     pub config: CheatsConfig,
+
+    /// Test-scoped context holding data that needs to be reset every test run
+    pub context: Context,
+}
+
+#[derive(Debug, Default)]
+pub struct Context {
+    //// Buffered readers for files opened for reading (path => BufReader mapping)
+    pub opened_read_files: HashMap<String, BufReader<File>>,
+}
+
+/// Every time we clone `Context`, we want it to be empty
+impl Clone for Context {
+    fn clone(&self) -> Self {
+        Default::default()
+    }
 }
```

Comment:
this is a suitable solution, because we will have a new,empty context for each test call.

I have some ideas on how to make this perhaps a bit more idiomatic in the multifork PR